forge-game/src/main/java/forge/game/Game.java

```diff
@@ -368,7 +402,7 @@ public final PlayerCollection getPlayersInTurnOrder(Player p) {
         PlayerCollection players = getPlayersInTurnOrder();
 
         int i = players.indexOf(p);
-        Collections.rotate(players, i);
+        Collections.rotate(players, -i);
```

Comment:
your failed test:

Question:
this implies the AI is making a pretty big misplay now, but why...?

Answer:
with default direction, you are **modifying the original order of the player slots**

This took me ages to find 💩 
(Luckily I noticed on your branch when loading gamestates the boards would get swapped around depending on which turns player you loaded it)

Reply:
Oh sweet. Good catch! Hopefully that's the last thing causing issues, so we can get people experimenting with it.